test/interpolation.jl

```diff
@@ -25,3 +28,33 @@ using LinearAlgebra
     ρout2 = interpolate_density(ρin, grid_in, grid_out, lattice, 4lattice)
     @test maximum(abs.(ρout2 - ρout1)) < .01
 end
+
+@testset "Interpolation of blochwave" begin
+    tol = 1e-7
+
+    Si = ElementPsp(silicon.atnum, psp=load_psp(silicon.psp))
+    model = model_LDA(silicon.lattice, [Si => silicon.positions])
+    kgrid = [2,2,2]
+    Ecut = 5
+    basis = PlaneWaveBasis(model, Ecut; kgrid=kgrid)
+
+    # Run nlsolve without guess
```

Comment:
Well with zero guess really ...

Reply:
I removed it, it was some copy-paste from other part of the code, but not very useful.